yarn-project/end-to-end/src/spartan/gating-passive.test.ts

```diff
@@ -135,16 +133,13 @@ describe('a test that passively observes the network in the presence of network
       await sleep(Number(epochDuration * slotDuration) * 1000);
       const newTips = await rollupCheatCodes.getTips();
 
-      // calculate the percentage of slots missed
+      // calculate the percentage of slots missed for debugging purposes
       const perfectPending = controlTips.pending + BigInt(Math.floor(Number(epochDuration)));
       const missedSlots = Number(perfectPending) - Number(newTips.pending);
       const missedSlotsPercentage = (missedSlots / Number(epochDuration)) * 100;
       debugLogger.info(`Missed ${missedSlots} slots, ${missedSlotsPercentage.toFixed(2)}%`);
 
-      // Ensure we missed at most the max allowed slots
-      // This is in place to ensure that we don't have a bad regression in the network
-      const maxMissedSlots = Math.floor(Number(epochDuration) * MAX_MISSED_SLOT_PERCENT);
-      expect(missedSlots).toBeLessThanOrEqual(maxMissedSlots);
+      expect(newTips.pending).toBeGreaterThan(controlTips.pending);
```

Comment:
When we have short epochs (e.g. 4), its very reasonable for us to only have 1 block get produced: the nodes that restart could be proposers for all 4 blocks, and it could take about a minute for them to come back online after the restart.